packages/fx-core/src/plugins/resource/bot/index.ts

```diff
@@ -208,7 +213,11 @@ export class TeamsBot implements Plugin {
       return await TeamsBot.runWithExceptionCatching(
         context,
         async () => {
-          if (isBotNotificationEnabled()) {
+          if (isVSProject(context.projectSettings) || context.answers?.platform === Platform.VS) {
+            const res = new QTreeNode(createHostTypeTriggerQuestionForVS());
```

Comment:
The logic between L216 and L219 looks the same as L221 and L229, so for the long term (after VS 17.3 preview 2), createHostTypeTriggerQuestionForVS() seems to be part of createHostTypeTriggerQuestion() logic since it already takes context.answers?.platform as param. @yukun-dong